app/utils/bundle-importer.js

```diff
@@ -460,7 +481,20 @@ YUI.add('bundle-importer', function(Y) {
               name = requiredModel.get('name') + '/' + size;
               break;
             case 2:
-              record.args[2] = requiredModel.id;
+              // Check whether this placement refers to a machine or a unit.
```

Comment:
Could you elaborate here on what's actually going on here. The documentation for this method is really poor because the code doesn't really 'self document' - I'll take the blame for that :)